packages/discovery-provider/src/api/v1/users.py

```diff
@@ -1607,6 +1608,64 @@ def get(self, id):
         return super()._get(id)
 
 
+mutual_followers_response = make_response(
+    "mutual_followers_response", ns, fields.List(fields.Nested(user_model))
+)
+full_mutual_followers_response = make_full_response(
+    "full_mutual_followers_response",
+    full_ns,
+    fields.List(fields.Nested(user_model_full)),
+)
+
+USER_MUTUAL_FOLLOWERS_ROUTE = "/<string:id>/mutuals"
+
+
+@full_ns.route(USER_MUTUAL_FOLLOWERS_ROUTE)
+class FullMutualFollowers(Resource):
```

Comment:
Wondering if we want to require auth for this? Or if we're okay just letting anyone list mutuals for any pair of users.
🤷